src/util/categories.ts

```diff
@@ -152,7 +152,7 @@ export async function trimUnmirroredPages(downloader: Downloader) {
                     if (processedArticles % 100 === 0) {
                         const percentProgress = Math.floor(processedArticles / numKeys * 1000) / 10;
                         if (percentProgress !== prevPercentProgress) {
-                            prevPercentProgress = percentProgress;
+                            prevPercentProgress = Math.min(percentProgress, 100);
```

Comment:
To me, this looks like a hack. the consequence of this is that the log will be "stuck" at 100% and the scrape will continue, which is not right either. The problem here is: why do we have her `processedArticles` which can be bigger than `numKeys`